queries/orderbook/barn_batch_rewards.sql

```diff
@@ -185,7 +172,7 @@ reward_data AS (
         ss.auction_id,
         -- TODO - Assuming that `solver == winner` when both not null
         --  We will need to monitor that `solver == winner`!
-        COALESCE(os.solver, winner) AS solver,
+        ss.winner AS solver,
```

Comment:
Just
```suggestion
        winner AS solver,
```
would also have worked.